extras/create_send_notifications.py

```diff
@@ -188,6 +196,11 @@ def main():
     else:
         logging.info("Nothing to email - skipping this step")
 
+    # Remove orgs from notifications_to_delete if they are in the list of orgs
+    # that we just generated notifications for (most likely because the
+    # notification was included in an ancestor org's notification)
+    notifications_to_delete = sorted(set(notifications_to_delete) - set(orgs_notified))
```

Comment:
Thoughts on changing the first use of `notifications_to_delete` to something like `notifications_not_generated` (and propagating this proposed name change to the same variable name in the `build_notifications_org_list()` function)? I'm generally not a fan of overwriting variables with derived values as it makes the original variable name slightly misleading.

Reply:
I like it!

Reply:
Please see https://github.com/cisagov/cyhy-reports/pull/104/commits/2e1377549d690b7d7899661bf9e01ea4eb2d2d54

Reply:
See also commit ee5fdc52520b51ae1c6878083494fa03bfdba1ba.

Reply:
Also https://github.com/cisagov/cyhy-reports/pull/104/commits/7d844b3fcad1b3ea7d20f038ce203303e03a5a9c for completeness.